src/components/calcite-slider/calcite-slider.e2e.ts

```diff
@@ -162,34 +165,35 @@ describe("calcite-slider", () => {
     `);
     const slider = await page.find("calcite-slider");
     const handle = await page.find("calcite-slider >>> .thumb");
+    const inputEvent = await slider.spyOnEvent("calciteSliderInput");
     const changeEvent = await slider.spyOnEvent("calciteSliderChange");
+    expect(inputEvent).toHaveReceivedEventTimes(0);
     expect(changeEvent).toHaveReceivedEventTimes(0);
     await handle.press("ArrowRight");
+    await page.waitForChanges();
+    expect(await slider.getProperty("value")).toBe(24);
+    expect(inputEvent).toHaveReceivedEventTimes(1);
     expect(changeEvent).toHaveReceivedEventTimes(1);
   });
 
   describe("mouse interaction", () => {
     it("single handle: clicking the track changes value on mousedown, emits on mouseup", async () => {
       const page = await newE2EPage({
-        html: `
-          <style>body { margin: 0; }</style>
-          <div style="width:100px">
-            <calcite-slider snap></calcite-slider>
-          </div>
-        `
+        html: `<calcite-slider snap style="width:${sliderWidthFor1To1PixelValueTrack}"></calcite-slider>`
       });
+      await page.waitForChanges();
       const slider = await page.find("calcite-slider");
       const changeEvent = await slider.spyOnEvent("calciteSliderChange");
```

Comment:
Can you make sure all tests that assert events spy on both the change and input events?

Reply:
added assertion for both input and change events.